{{ cookiecutter.package_name }}/tox.ini

```diff
@@ -0,0 +1,87 @@
+[tox]
+envlist =
+    py{36,37,38}-test{,-alldeps,-devdeps}
+    py{36,37,38}-test-numpy{116,117,118}
+    py{36,37,38}-test-astropy{30,40,lts}
+    build_docs
+    codestyle
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+isolated_build = true
+
+[testenv]
+
+# Pass through the following environemnt variables which may be needed for the CI
+passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI TRAVIS
+
+# Run the tests in a temporary directory to make sure that we don't import
+# astropy from the source tree
+changedir = .tmp/{envname}
+
+# tox environments are constructued with so-called 'factors' (or terms)
+# separated by hyphens, e.g. test-devdeps-cov. Lines below starting with factor:
+# will only take effect if that factor is included in the environment name. To
+# see a list of example environments that can be run, along with a description,
+# run:
+#
+#     tox -l -v
+#
+description =
+    run tests
+    alldeps: with all optional dependencies
+    devdeps: with the latest developer version of key dependencies
+    oldestdeps: with the oldest supported version of key dependencies
+    cov: and test coverage
+    numpy116: with numpy 1.16.*
+    numpy117: with numpy 1.17.*
+    numpy118: with numpy 1.18.*
+    astropy30: with astropy 3.0.*
+    astropy40: with astropy 4.0.*
+    astropylts: with the latest astropy LTS
+
+# The following provides some specific pinnings for key packages
+deps =
+
+    numpy116: numpy==1.16.*
+    numpy117: numpy==1.17.*
+    numpy118: numpy==1.18.*
+
+    astropy30: astropy==3.0.*
+    astropy40: astropy==4.0.*
+    astropylts: astropy==4.0.*
+
+    devdeps: git+https://github.com/numpy/numpy.git#egg=numpy
+    devdeps: git+https://github.com/astropy/astropy.git#egg=astropy
+
+# The following indicates which extras_require from setup.cfg will be installed
+extras =
+    test
+    alldeps: all
+
+commands =
+    pip freeze
+    pytest --pyargs {{ cookiecutter.module_name }} {toxinidir}/docs --cov {{ cookiecutter.module_name }} --cov-config={toxinidir}/setup.cfg {posargs}
```

Comment:
Of course, coverage is listed here, so it always happens.

Reply:
I guess it needs the following from astropy's tox.ini:
```
    !cov: pytest --pyargs packagename {toxinidir}/docs {posargs}
```

Reply:
Done